shell-completion/bash/homectl

```diff
@@ -50,7 +50,7 @@ _homectl() {
                         --realm
                         --email-address
                         --location
-                        --icon-name
+                        --avatar-path
```

Comment:
why drop the icon name?

Reply:
At the time it was being replaced w/ avatarPath. And I'm really not convinced anything (gnome-shell, GDM, etc) will ever display the iconName